src/ui/components/OverlayCard/index.js

```diff
@@ -47,6 +39,7 @@ export default class OverlayCard extends React.Component {
     return (
       <Overlay
         onEscapeOverlay={this.props.onEscapeOverlay}
+        id={id}
         visibleOnLoad={visibleOnLoad}
         ref={(ref) => {
```

Comment:
You can just remove all the refs, I think

Reply:
will do : )

Reply:
Actually if I do this, I will have to update the OverlayCard (since it's using these refs in its tests). I guess I was trying to avoid that in this PR at least..
I am still having that store issue with this file so I may have to anyway tho :/
I'll follow up : )